.github/workflows/ci.yml

```
@@ -126,4 +126,45 @@ jobs:
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
-        run: sbt --client '++${{ matrix.scala }}; ci-release'
\ No newline at end of file
+        run: sbt --client '++${{ matrix.scala }}; ci-release'
+
+  site:
+    name: Deploy site
+    needs: [publish]
+    if: needs.build.result == 'success' && (needs.publish.result == 'success' && startsWith(github.ref, 'refs/tags/v'))
```

It doesn't even define a `build` need. And anyway, GH will only execute if the needs were successful, so:
```suggestion
    if: startsWith(github.ref, 'refs/tags/v')
```